Author: SVN-Git Migration <python-modules-team@lists.alioth.debian.org>
Date: Thu, 8 Oct 2015 13:13:30 -0700
Description: Disable domain tests to prevent internet access during build
Forwarded: Debian-specific

---
 tornado/test/netutil_test.py | 1 +
 1 file changed, 1 insertion(+)

diff --git a/tornado/test/netutil_test.py b/tornado/test/netutil_test.py
index 32807e2..e6a0c7e 100644
--- a/tornado/test/netutil_test.py
+++ b/tornado/test/netutil_test.py
@@ -49,6 +49,7 @@ class _ResolverTestMixin(object):
 # resolution, so test this case separately, using mocks as needed.
 class _ResolverErrorTestMixin(object):
     @gen_test
+    @unittest.skip("Prevent internet access during build")
     def test_bad_host(self):
         with self.assertRaises(IOError):
             yield self.resolver.resolve("an invalid domain", 80, socket.AF_UNSPEC)
